# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/TestGroupId.cpp
# Commit: eb376fd954e1
# Full Hash: eb376fd954e1287b8e4f504f743b7159f1b514d0
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1651745
# File Overlap Count: 1
# Description:
#   Bug 1651745 - Change MediaEngineSource::SetTrack to take a MediaTrack base class. r=padenot
#   
#   This allows us to pass in different sub classes, which the next patch will do.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95933
# ==============================================================================

diff -r 2799805eef3b -r eb376fd954e1 dom/media/gtest/TestGroupId.cpp
--- a/dom/media/gtest/TestGroupId.cpp	Thu Nov 05 16:41:45 2020 +0000
+++ b/dom/media/gtest/TestGroupId.cpp	Thu Nov 05 16:42:07 2020 +0000
@@ -37,7 +37,7 @@
                nsresult(const dom::MediaTrackConstraints&,
                         const MediaEnginePrefs&, uint64_t, const char**));
   MOCK_METHOD2(SetTrack,
-               void(const RefPtr<SourceMediaTrack>&, const PrincipalHandle&));
+               void(const RefPtr<MediaTrack>&, const PrincipalHandle&));
   MOCK_METHOD0(Start, nsresult());
   MOCK_METHOD3(Reconfigure, nsresult(const dom::MediaTrackConstraints&,
                                      const MediaEnginePrefs&, const char**));